fixtures/u2_func_simple.rb.sem.uast

```diff
@@ -0,0 +1,159 @@
+                              },
+                              target: { '@type': "uast:Identifier",
+                                 '@role': [Binary, Left],
+                                 '@pos': { '@type': "uast:Positions",
+                                    start: { '@type': "uast:Position",
+                                       offset: 40,
+                                       line: 3,
+                                       col: 12,
+                                    },
+                                    end: { '@type': "uast:Position",
+                                       offset: 40,
+                                       line: 3,
+                                       col: 12,
+                                    },
+                                 },
+                                 Name: "a",
+                              },
+                           },
+                        ],
+                     },
+                  ],
+               },
+               Type: { '@type': "uast:FunctionType",
+                  Arguments: [
+                     { '@type': "uast:Argument",
+                        '@pos': { '@type': "uast:Positions",
+                           start: { '@type': "uast:Position",
+                              offset: 13,
+                              line: 1,
+                              col: 14,
+                           },
+                           end: { '@type': "uast:Position",
+                              offset: 13,
+                              line: 1,
+                              col: 14,
+                           },
+                        },
+                        Init: ~,
+                        MapVariadic: false,
+                        Name: { '@type': "uast:Identifier",
+                           Name: "a",
+                        },
+                        Receiver: false,
+                        Type: ~,
+                        Variadic: false,
+                     },
+                     { '@type': "uast:Argument",
+                        '@pos': { '@type': "uast:Positions",
+                           start: { '@type': "uast:Position",
+                              offset: 16,
+                              line: 1,
+                              col: 17,
+                           },
+                           end: { '@type': "uast:Position",
+                              offset: 16,
+                              line: 1,
+                              col: 17,
+                           },
+                        },
+                        Init: ~,
+                        MapVariadic: false,
+                        Name: { '@type': "uast:Identifier",
+                           Name: "b",
+                        },
+                        Receiver: false,
+                        Type: ~,
+                        Variadic: false,
+                     },
+                  ],
+                  Returns: ~,
```

Comment:
Ruby function always has a single return (implicit `Null`)

Reply:
If we used implicit Null as a default type but the function returned other types (implicitly or with a return), it would be wrong. To know the exact type we would have to "interpret" the AST just like the Ruby interpreter does, so I think leaving it empty is the less bad option.

Reply:
I'm not talking about implicit `Null` as a return type. I meant that Ruby functions always have a return of unknown type (`nil` type in `uast:Argument`). But it's always there, regardless if you `return` something or not.

Reply:
Oh, I think I understand now, you are saying that languages where we can't get the return type from the function declaration should have a nil in the `Returns` field, right? But isn't the `~` the way the serialization format writes `nil`s?

Reply:
Nil in `Returns` is different from a nil return type. Nil in returns means the output parameter array is empty, thus the function returns exactly 0 values (aka void). But this is not what happens in Ruby, so I propose to always return an array with one `uast:Argument` node that has a nil `Type`, since we can't infer type for now. You can check how JS [does this](https://github.com/bblfsh/javascript-driver/pull/20/files#diff-da412cf66633e782d4653e9a0a1b54bdR207) since it has similar semantics.